app/controllers/admin/api_docs/service_api_docs_controller.rb

```diff
@@ -6,9 +6,21 @@ class Admin::ApiDocs::ServiceApiDocsController < Admin::ApiDocs::BaseController
   activate_menu :serviceadmin, :ActiveDocs
   sublayout 'api/service'
 
+  def index
+    @api_docs_services = current_scope.all_api_docs.page(params[:page])
```

Comment:
```suggestion
    @api_docs_services = current_scope.api_docs.page(params[:page])
```

Will `Service` respond to `all_api_docs` too?